botocore/handlers.py

```diff
@@ -781,11 +781,25 @@ def _replace_content(self, section):
         section.write(updated_content)
 
 
+class CommandAlias(object):
```

Comment:
Sort of a strange class name because of the name ``Command`` since there is no command abstraction. Maybe call this ``ClientMethodAlias``?